iaso/api/entity.py

```diff
@@ -80,6 +83,12 @@ def get_submitter(self, entity: Entity):
     def get_duplicates(self, entity: Entity):
         return _get_duplicates(entity)
 
+    def get_nfc_cards(self, entity: Entity):
+        nfc_devices = StorageDevice.objects.filter(entity=entity, type=StorageDevice.NFC)
+        nfc_log_entries_count = StorageLogEntry.objects.filter(device__in=nfc_devices).count()
+
+        return nfc_log_entries_count
+
```

Comment:
I think we should be careful to add this extra query on the EntitySerializer. I also expect the amount of records in `StorageLogEntry` to grow.
 Did you try to do it with a relation and `prefetch_related`? If it's complicated then it's okay for me to go with it like this.